ddprofiler/app/src/main/java/ddprofiler/analysis/TextualAnalyzer.java

```diff
@@ -7,26 +7,26 @@
 import java.util.Iterator;
 import java.util.List;
 
-import ddprofiler.analysis.modules.Cardinality;
-import ddprofiler.analysis.modules.CardinalityAnalyzer;
-import ddprofiler.analysis.modules.Entities;
-import ddprofiler.analysis.modules.EntityAnalyzer;
-import ddprofiler.analysis.modules.KMinHash;
+import ddprofiler.analysis.modules.*;
```

Comment:
would be better to avoid the wildcard (*) here and flesh out the classes to import